sdk/search/Azure.Search.Documents/src/autorest.md

```diff
@@ -191,6 +191,53 @@ directive:
     $.properties.endTime["x-nullable"] = true;
 ```
 
+#### Skills
+
+``` yaml
+directive:
+- from: swagger-document
+  where: $.definitions.EntityRecognitionSkill
+  transform: >
+    $.properties.defaultLanguageCode["x-nullable"] = true;
+
+- from: swagger-document
+  where: $.definitions.ImageAnalysisSkill
+  transform: >
+    $.properties.defaultLanguageCode["x-nullable"] = true;
+
+- from: swagger-document
+  where: $.definitions.KeyPhraseExtractionSkill
+  transform: >
+    $.properties.defaultLanguageCode["x-nullable"] = true;
```

Comment:
So this usually doesn't work because we're using different properties, but I think there's a shortcut you can use to collapse some of these if you felt so inclined.

```yaml
- from: swagger-document
  where:
  - $.definitions.EntityRecognitionSkill
  - $.definitions.ImageAnalysisSkill
  - $.definitions.KeyPhraseExtractionSkill
  transform: >
    $.properties.defaultLanguageCode["x-nullable"] = true;
```

Reply:
Thanks. Good to know, but I think I'll skip this and keep it organized by model type. Might help with getting the swagger fixed.